projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -35,6 +36,21 @@ public function register_routes() {
 					'methods'             => WP_REST_Server::READABLE,
 					'callback'            => array( $this, 'get_data' ),
 					'permission_callback' => array( $this, 'can_access' ),
+					'args'                => array(
+						'checklist_slug' => array(
+							'description' => 'Checklist slug',
+							'type'        => 'string',
+							'enum'        => array(
```

Comment:
We will want to build this list directly from the Checklists registry. Again, this is fine for now, but let's circle back later.

Reply:
I've ensured follow up notes for both the require_once and for building the list of args from the checklist registry are adding to our tracking task for the Launchpad refactor.